src/wallet/rpcwallet.cpp

```diff
@@ -2440,6 +2440,9 @@ static RPCHelpMan getwalletinfo()
                             {RPCResult::Type::NUM, "progress", "scanning progress percentage [0.0, 1.0]"},
                         }},
                         {RPCResult::Type::BOOL, "descriptors", "whether this wallet uses descriptors for scriptPubKey management"},
+#ifdef ENABLE_EXTERNAL_SIGNER
+                        {RPCResult::Type::BOOL, "external_signer", "whether this wallet uses an external signer such as a hardware wallet"},
```

Comment:
Not sure about having this as a compile-time conditional. Maybe document it and mention that omission means the build doesn't support it?

Reply:
I'm dropping the `#ifdef` here and below.  The `WALLET_FLAG_EXTERNAL_SIGNER` is always defined anyway.